hibernate-spatial/src/main/java/org/hibernate/spatial/dialect/postgis/PGSpatialDialectTrait.java

```diff
@@ -125,4 +127,13 @@ default boolean supportsFiltering() {
 	default boolean supports(SpatialFunction function) {
 		return support.supports( function );
 	}
+
+	/**
+	 * Checks whether the typeCode is (potentially) the code for a spatial type
+	 * @param typeCode the JDBC type code
+	 * @return if the typecode corresponds with a spatialt type
```

Comment:
I did some research here. So there is some subtle difference between 'corresponds to' and 'corresponds with'. Seems 'corresponds to' is more appropriate here for 'corresponds with' means 'comply with' or 'in harmony'.